irepela/homework_2/fizz_buzz.py

```diff
@@ -0,0 +1,9 @@
+def checkio(number):
+    if number % 3 == 0 and number % 5 == 0:
+        return "Fizz Buzz"
+    elif number % 3 == 0:
+        return "Fizz"
+    elif number % 5 == 0:
+        return "Buzz"
+    else:
+        return str(number)
```

Comment:
I would suggest reducing return keyword usage to minimum,

since in such case you need to be sure that each branch contains return
in such cases I prefer such structure


```
def checkio(num):
    res  = str(num)
   
    if:
         ...
    elif:
         ...
    elif:
        ...
 
    return res



```